### PR TITLE
msklc: Add version 1.4

### PR DESCRIPTION
**Pull request by sitiom**
- The only available tool (aside from [KbdEdit](http://kbdedit.com/)) to create and modify keyboard layouts on Windows
- Abandonware since 2007, no autoupdate

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/msklc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "Expand-MSIArchive \"$dir\\MSKLC.msi\" -Removal",
+            "Remove-Item \"$dir\\setup.exe\""
+        ]
+    },
+    "bin": "bin\\i386\\kbdutool.exe",
+    "shortcuts": [
+        [
+            "MSKLC.exe",
+            "Microsoft Keyboard Layout Creator 1.4"
+        ]
```
Comment (issaclin32) [inline bucket/msklc.json:20]:
```suggestion
        [
            "MSKLC.exe",
            "Microsoft Keyboard Layout Creator"
        ]
```

I think the version number is not needed.

Comment (sitiom) [reply bucket/msklc.json:20]: The installer does that, but I guess we could "modernize" the name.

**Comment by issaclin32** [comment]
config location: registry `HKCU:\SOFTWARE\Microsoft\Keyboard Layout Creator 1.4`.
(This is a reminder. No need to modify anything regard to this.)

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**